macOS/DuckDuckGo/WebExtensions/WebExtensionManager.swift

```diff
@@ -123,7 +124,7 @@ final class WebExtensionManager: NSObject, WebExtensionManaging {
     }
 
     var hasInstalledExtensions: Bool {
-        controller.extensions.count > 0
+        pathsCache.cache.count > 0
```

Comment:
I don't understand this change.  Are we counting path cache entries or installed extensions?

To me it sounds like the right source of truth should be controller.extensions, which is basically the installed extensions.